doc/design/architecture.md

```diff
@@ -11,17 +11,19 @@ Each of these Operators is responsible for managing the CRDs that are the basis
 | CatalogSource         | catsrc         | Catalog | a repository of CSVs, CRDs, and packages that define an application                        |
 | Subscription          | sub        | Catalog | used to keep CSVs up to date by tracking a channel in a package                            |
 | OperatorGroup         | og     | OLM     | used to group multiple namespaces and prepare for use by an operator                     |
+| PackageManifest       |        | PackageServer | provides the user with an api presentation of the data a CatalogSource provides |
```

Comment:
nit: PackageManifest isn't actually a CRD, it's just a synthetic aggregated api resource. You can't, for example, create a PackageManifest CR and have it stored in etcd. We should probably not put it in the list of CRDs.